scripts/templates/component/Component.test.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+import Component from '.';
+
+describe('Component', () => {
```

Comment:
Sorry I am whining about this right now, but aren't we using double quotes in dcos-ui? Would love to have it the same